### PR TITLE
Accessibility updates

### PR DESCRIPTION
**Pull request by davidmurdoch**
This PR attempts to implement https://github.com/trufflesuite/dev-handbook/issues/28

This PR makes the website keyboard and screenreader friendly. It increase contrasts where necessary and makes other changes to attempt to make the website more useful for low-vision users. Additionally, small changes were made to layout where previously things may have been hard to see.

NOte: Dark Mode failed accessibility so greatly that I have disabled it in this PR 😢 .

### CODE REVIEW

File: custom_theme/ganache.html [hunk 1]
```diff
@@ -131,52 +123,52 @@ <h2>ONE CLICK BLOCKCHAIN</h2>
                 <br/>
                 <span class="os">(Windows)</span>
                 </p>
-                <img src="/img/ganache/windows-logo.svg" />
+                <img alt="" src="/img/ganache/windows-logo.svg" />
             </a>
-            <p class="ganache-other-os first"><a href="https://github.com/trufflesuite/ganache-ui/releases">Need another OS download?</a></p>
+            <p class="ganache-other-os first"><a target="_blank" rel="noopener noreferrer" href="https://github.com/trufflesuite/ganache-ui/releases">Need another OS download?</a></p>
```
Comment (eggplantzzz) [inline custom_theme/ganache.html:128]: it's funny that it is `noreferrer` - I think the actual field is the misspelling, "referer"

File: src/accessibility/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+template: template-no-sidebar.html
+title: Accessibility
+---
+
+# Accessibility
+
+Truffle is commited to providing a website that is accessible to the widest possible audience,
```
Comment (eggplantzzz) [inline src/accessibility/index.md:8]: "committed" and not "commited"

File: src/case-studies/case-study-dish-network.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ---
 template: case-study-dish-network.html
-title: Truffle Suite
+title: Case Study - Dish Network
+description: Truffle, though its Truffle University program, taught engineers at DISH how to build blockchain applications.
```
Comment (eggplantzzz) [inline src/case-studies/case-study-dish-network.md:4]: "through" not "though"

File: src/drizzle/index.md [hunk 4]
```diff
@@ -1,4 +1,5 @@
 ---
 template: drizzle.html
-title: Truffle Suite
+title: Truffle Drizzle
```
Comment (eggplantzzz) [inline src/drizzle/index.md:3]: Should this just be "Drizzle"?

**Review by eggplantzzz — APPROVED** [review]

File: custom_theme/ganache.html [hunk 5]
```diff
@@ -131,52 +123,52 @@ <h2>ONE CLICK BLOCKCHAIN</h2>
...
             <h2 class="mt-4">VISUAL MNEMONIC &amp; ACCOUNT INFO</h2>
             <p>Quickly see the current status of all accounts, including their addresses, private keys, transactions and balances.</p>
             </div>
         </div>
-        
+
         <div class="row text-center mb-5 mt-5">
             <div class="col-md-6 mb-5 mb-md-0">
-            <img class="feature-icon-sm mb-4" src="/img/ganache-feature-3-icon.svg" alt="Ganache" title="Ganache" />
+            <img class="feature-icon-sm mb-4" src="/img/ganache-feature-3-icon.svg" alt=""" />
```
Comment (davidmurdoch) [inline custom_theme/ganache.html:146]:
```suggestion
            <img class="feature-icon-sm mb-4" src="/img/ganache-feature-3-icon.svg" alt="" />
```

**Review by xhulz — APPROVED** [review]

File: src/accessibility/index.md (hunk 2 above)
Comment (davidmurdoch) [inline src/accessibility/index.md:8]:
```suggestion
Truffle is committed to providing a website that is accessible to the widest possible audience,
```

File: src/accessibility/index.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+title: Accessibility
+---
+
+# Accessibility
+
+Truffle is commited to providing a website that is accessible to the widest possible audience,
+regardless of technology or ability. We are actively working to increase the accessibility and
+usability of our website and the developer tools we create.
+
+We endeavour to conform to level Double-A of the World Wide Web Consortium (W3C) Web Content
```
Comment (davidmurdoch) [inline src/accessibility/index.md:12]:
```suggestion
We endeavor to conform to level Double-A of the World Wide Web Consortium (W3C) Web Content
```

File: src/case-studies/case-study-microsoft-xbox.md [hunk 7]
```diff
@@ -1,4 +1,5 @@
 ---
 template: case-study-microsoft-xbox.html
-title: Truffle Suite
+title: Case Study - Microsoft Xbox
+description: Microsoft and EY partnered to develop royalty payment system for Xbox using Truffle and the Quorum blockchain.
```
Comment (davidmurdoch) [inline src/case-studies/case-study-microsoft-xbox.md:4]:
```suggestion
description: Microsoft and EY partnered to develop a royalty payment system for Xbox using Truffle and the Quorum blockchain.
```

File: src/drizzle/index.md (hunk 4 above)
Comment (davidmurdoch) [inline src/drizzle/index.md:3]:
```suggestion
title: Drizzle
```

File: src/ganache/index.md [hunk 8]
```diff
@@ -1,4 +1,5 @@
 ---
 template: ganache.html
-title: Truffle Suite
+title: Truffle Ganache
```
Comment (davidmurdoch) [inline src/ganache/index.md:3]:
```suggestion
title: Ganache
```

**Pull request merged**